build-logic-settings/settings.gradle.kts

```diff
@@ -22,6 +22,8 @@ dependencyResolutionManagement {
 }
 
 plugins {
+    id("com.gradle.develocity").version("3.17.5") // Sync with `build-logic-commons/build-platform/build.gradle.kts`
```

Comment:
❌ all three plugins should either use `.version(...)` or ` version(...)` consistently

Reply:
Can we make sure we point to this location from the other two as well? If not, we'll forget it when upgrading the DV plugin.